src/partials/exercises.html

```diff
@@ -7,7 +7,7 @@
 -->
 
 <section class="exersises container">
-    <div class="content-filter-container">
+    <!-- <div class="content-filter-container">
```

Comment:
If this code will be in this section then it will be only over right column and not full width